crates/eigen-cli/src/args.rs

```diff
@@ -42,4 +42,70 @@ outputs addresses for all relevant Eigenlayer and AVS contracts within the netwo
         #[arg(long, help = "rpc url", default_value = ANVIL_RPC_URL)]
         rpc_url: String,
     },
+
+    #[command(about = "EigenLayer CLI Key tools", name = "egnkey")]
+    EigenKey {
+        #[command(subcommand)]
+        subcommand: EigenKeyCommand,
+    },
+}
+
+#[derive(Subcommand, Debug)]
+pub enum EigenKeyCommand {
+    #[command(
+        about = "Generate keys for testing purpose.
+This command creates ecdsa or bls key pair for testing purposes. It generates
+all the relevant files for reading and keys and decrypted it and also gets 
+you the private keys in plaintext.
+
+It creates the following artifacts based on arguments
+- passwords.txt - contains all passwords to decrypt keys
+- private_key_hex.txt - will create plaintext private keys
+- keys/* - create all the encrypted json files in this folder",
+        alias = "g"
+    )]
+    Generate {
+        #[arg(long, help = "key type to create (ecdsa or bls)")]
+        #[clap(value_enum)]
+        key_type: KeyType,
+
+        #[arg(long, help = "number of keys to create", default_value = "1")]
+        num_keys: u32,
+
+        #[arg(long, help = "folder to store keys")]
+        output_dir: Option<String>,
+    },
+
+    #[command(
+        about = "Given a private key, output its associated operatorId (hash of bn254 G1 pubkey).",
+        alias = "d"
+    )]
+    DeriveOperatorId {
+        #[arg(
+            long,
+            help = "(bn254) private key from which to derive operatorId from"
+        )]
+        private_key: String,
+    },
+
+    #[command(
+        about = "Stores an ecdsa key to a file, in web3 secret storage format.",
+        alias = "c"
+    )]
+    Convert {
```

Comment:
can we name this as something ConvertECDSA? since i am working on a keystore integartion for bls also, it will be confusing

Reply:
Done!